testing/core/tests/BUILD.bazel

```diff
@@ -31,6 +31,8 @@ expand_location_unit_test_suite()
         "expr-attribute-test",
         "relative-imports",
         "extra-args-test",
+        "flake-hello",
+        "flake-hello-with-build-file",
```

Comment:
In reply to https://github.com/tweag/rules_nixpkgs/pull/380#issuecomment-1543902257.
The issue is that these tests were added here. This list comprehension is only valid for tests of packages that are imported using a bzlmod Bazel module extension. However, no such extension exists for the flake part, yet.
You can fix this by adding another instance of the list comprehension like this (unverified code):
```
[
    # All of these tests use the "hello" binary to see
    # whether different invocations of `nixpkgs_flake_package`
    # produce a valid bazel repository.
    sh_test(
        name = "run-{0}".format(test),
        timeout = "short",
        srcs = ["test_bin.sh"],
        args = ["$(location @{0}//:bin)".format(test)],
        data = ["@{0}//:bin".format(test)],
    )
    for test in [
        "flake-hello",
        "flake-hello-with-build-file",
```
(If you go through the Git history of this file, you'll find instances of such code from when not all tests had been converted to use the module extension, yet)

You'll also need to add `use_repo` entries to `MODULE.bazel` to import these two repositories from the `non_module_deps` extension.

Reply:
Is that just a temporary workaround or the proper fix?  It seems like my work here is only half done, as I presumably need to write `@rules_nixpkgs_core//extensions:flake_package.bzl` (or whatever bikeshed)?  This is based on my observation that `nix_pkg = use_extension("@rules_nixpkgs_core//extensions:package.bzl", "nix_pkg")` seems to be a bzlmod wrapper/validator around `nixpkgs_package()`.

Reply:
bzlmod test fixes pushed.  Not sure if they work as the CI jobs don't seem to run automatically?

Reply:
Yeah, we have to run them manually for contributors from outside the organization. Looks like CI is passing now.